docs/podman-system-migrate.1.md

```diff
@@ -11,6 +11,12 @@ podman\-system\-migrate - Migrate container to the latest version of podman
 
 **podman system migrate** takes care of migrating existing containers to the latest version of podman if any change is necessary.
 
+Rootless Podman uses a pause process to keep the unprivileged namespaces alive.  This prevents any change to the `/etc/subuid`
+and `/etc/subgid` files to be propagated to the rootless containers when the pause process is already running.
+For these changes to be honored, it is necessary to stop all running containers and also stop the pause process.  Instead of doing it
+manually. **podman system migrate** can be used to stop the running containers and recreate the user namespace with the new
+configured mappings.
+
```

Comment:
Suggest:
"Rootless Podman uses a pause process to keep the unprivileged namespaces alive.  This prevents any change to the `/etc/subuid` and `/etc/subgid` files from being propagated to the rootless containers while the pause process is running.
For these changes to be propagated, it is necessary to first stop all running containers associated with the user and to also stop the pause process.  Instead of doing it manually, **podman system migrate** can be used to stop both the running containers and the pause process.  The `/etc/subuid` and `/etc/subgid` files can then be edited or changed with `usermod` to recreate the user namespace with the newly configured mappings."

Then add usermod (8) to the list of manpages below

Reply:
fixed now, thanks for the suggestion!